test_schemas/en/test_textfield_suggestions.json

```diff
@@ -36,15 +36,38 @@
                                         "id": "country-answer",
                                         "label": "What is your country of birth?",
                                         "mandatory": false,
-                                        "suggestions_url": "https://cdn.eq.census-gcp.onsdigital.uk/data/v5.0.0/gb/en/countries-of-birth.json",
+                                        "suggestions": {
+                                            "url": "https://cdn.eq.census-gcp.onsdigital.uk/eq-lookup-suggestions-data/v5.3.0/gb/en/countries-of-birth.json",
+                                            "allow_multiple": false
+                                        },
                                         "type": "TextField"
                                     }
                                 ],
                                 "id": "country-of-birth-question",
                                 "title": "Title",
                                 "type": "General"
-                            },
-                            "routing_rules": []
+                            }
+                        },
+                        {
+                            "type": "Question",
+                            "id": "country-block-multiple-selections",
+                            "question": {
+                                "answers": [
+                                    {
+                                        "id": "country-multiple-selections-answer",
+                                        "label": "What is your country of birth?",
```

Comment:
It was already wrong in this schema, but we should change it now. "What is your country of birth?" should be the question text and this should be something like "Country". This should also be fixed for the first question.

Reply:
Fixed now, thanks!